.github/workflows/build.yaml

```diff
@@ -0,0 +1,26 @@
+name: Build
+on:
+  push:
+    tags:
+      - v*
```

Comment:
do you need a prefix? Traditionally we didn't have a prefix. And for example, Poetry himself does not use the prefix

Reply:
This is the special prefix for poetry-dynamic-versioning used to identify version tags. The prefix will not be used in versioning.

Reply:
It is possible to set a custom pattern if it matters.

Reply:
may be set?

```
[tool.poetry-dynamic-versioning]
pattern = "(?x)^(?P<base>\\d+\\.\\d+\\.\\d+)(-?((?P<stage>[a-zA-Z]+)\\.?(?P<revision>\\d+)?))?(\\+(?P<tagged_metadata>.+))?$"
```

Reply:
Done.